BraintreeCore/src/main/java/com/braintreepayments/api/core/UserCanceledException.kt

```diff
@@ -14,7 +14,7 @@ import androidx.annotation.RestrictTo
  * could mean the user returned to the app through multi-tasking without completing the flow,
  * the user closed the browser tab, or the user pressed the back button.
  */
-open class UserCanceledException @JvmOverloads @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) constructor(
+class UserCanceledException @JvmOverloads @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) constructor(
     message: String?,
-    val isExplicitCancelation: Boolean = false
+    val isExplicitCancellation: Boolean = false
```

Comment:
```suggestion
    val isExplicitCancellation: Boolean = false
```
We actually use the [single 'L' variant](https://www.merriam-webster.com/grammar/canceled-or-cancelled) because 🇺🇸 .

Reply:
Oh but wait with "cancellation" it's double 'L' in [both languages](https://www.scribbr.com/us-vs-uk/cancelled-or-canceled/#cancelation)? 🤯

Reply:
the Android Studio spell check actually flagged it as a typo which is why I made the change. I was also seeing recommendations to avoid using the single L even in the US 🤔 

in any case, I think we can remove this property altogether!

Reply:
Reverted for this PR!